### PR TITLE
EOS-20348: Configurations to support CSM should run on management IP on port 80/443

### PR DESCRIPTION
**Pull request by soniyamoholkar**
# Backend

# Problem Statement
https://jts.seagate.com/browse/EOS-20348 

- _Overview: JIRA number/GitHub Issue added to PR: [Y/N]:_ Y
- _Describe the problem this patch intends to solve._
This patch has the changes related to configurations to enable the csm_web service on management IP on 443 port.

## Design

- _For Bug describe the fix here._ NA
- _For feature, post the link to the solution page._ NA
- _Any northbound interface change and has been notified to other gatekeepers [Y/N]:_ N

## Coding

- _Coding conventions are followed and code is consistent. Yes/No?_ Yes
- _Confirm All CODACY errors are resolved. Yes/No?_ Yes

## Testing

- _Confirm that Test Cases are added (for both the cases, fix and feature). Yes/No?_ Yes
- _Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability. Yes/No?_ Yes
- _Confirm Testing was performed with installed RPM. Yes/No?_ Yes
- _Confirm all the ut/regression/smoke test has been performed [Y/N]_ Y
- http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/cortx-manager/611/
- 
image

image

## Checklist

- _PR is self-reviewed? Yes/No?_ Yes
- _GitHub Issue is updated_ Yes
- _Jira is updated_
  -  _Check if the description is clear and explained._ 
    -  _Check Acceptance Criterion is defined._
      -  _All the tests performed should be mentioned before Resolving a JIRA._
        -  _Verification needs to be done before marked as Closed/Verified_
        - _Any interface change Yes/No?_ No
        - _Change notified to other gatekeepers: Yes/No?_ Yes
        - _Code reviews done and addressed: Yes/No?_ Yes
        - _Codacy issues reviewed and addressed: Yes/No?_ Yes
        - _Deployment test : Done/Not?_ 
        - _UT/smoke test: Done/Not?_ Yes
        - _Jira number added to PR: Yes/No?_ Yes
        - _Github merge done using UI: Yes/No?_ 
        - _Side effects on other features - deployment/update/node-replacement_ NA
        - _Changes to quick-start-guide/community impact_ NA
        - _All dependent component code PR are raised or already merged Yes/No_ NA        - 
        - _All dependent code already merged in landing branch Yes/No_ NA

### CODE REVIEW

File: csm/conf/configure.py [hunk 1]
```diff
@@ -166,15 +167,19 @@ def _logrotate(self):
...
-
-    def _configure_uds_keys(self):
-        Log.info("Configuring UDS keys")
+    
+    def _fetch_management_ip(self):
         cluster_id = Conf.get(const.CONSUMER_INDEX, self.conf_store_keys[const.KEY_CLUSTER_ID])
         virtual_host_key = f"{const.CLUSTER}>{cluster_id}>{const.NETWORK}>{const.MANAGEMENT}>{const.VIRTUAL_HOST}"
         self._validate_conf_store_keys(const.CONSUMER_INDEX,[virtual_host_key])
         virtual_host = Conf.get(const.CONSUMER_INDEX, virtual_host_key)
+        return virtual_host
```
Comment (udayan-y21) [inline csm/conf/configure.py:176]: Add managment  ip connectivity  validation check

**Review by udayan-y21 — CHANGES_REQUESTED** [review]

**Review by udayan-y21 — CHANGES_REQUESTED** [review]
Do we need to add modifications for .env file at the time of Reset/Cleanup phase? Because we may need to remove the IPs and need to reset the  .env file. Confirm with @kupranay  and @pawankumarsrivastava

File: csm/conf/configure.py (hunk 1 above)
Comment (kupranay) [reply csm/conf/configure.py:176]: Management IP Validation is note required

**Comment by soniyamoholkar** [comment]
> Do we need to add modifications for .env file at the time of Reset/Cleanup phase? Because we may need to remove the IPs and need to reset the .env file. Confirm with @kupranay and @pawankumarsrivastava

This will be taken care in another story https://jts.seagate.com/browse/EOS-21220

**Review by udayan-y21 — APPROVED** [review]

**Review by pawankumarsrivastava — APPROVED** [review]

**Review by kupranay — APPROVED** [review]

**Pull request merged**